src/grid/atomgrid.py

```diff
@@ -125,11 +123,17 @@ def quick_grid(
             different accuracy for atomic grid
             include: 'coarse', 'medium', 'fine', 'veryfine', 'ultrafine', 'insane'
         center : np.ndarray(3,), optional
+            if not given, default (0., 0., 0.) is used
             coordinates of grid center
         rotate : bool or int , optional
             Flag to set auto rotation for atomic grid, if given int, the number
             will be used as a seed to generate rantom matrix.
         """
+        center = (
+            np.zeros(3, dtype=float)
+            if center is None
+            else np.asarray(center, dtype=float)
+        )
         cls._input_type_check(rgrid, center)
         # load rad and npt
```

Comment:
It would be more helpful to be not use variable names in comment, especially when they are abbreviations.
So, here, you are loading radii & number of points?